ui/src/theme.js

```diff
@@ -335,6 +335,42 @@ const createTheme = isDarkTheme => {
           },
         },
       },
+      MuiPickersClock: {
+        pin: {
+          backgroundColor: THEME.SECONDARY,
+        },
+      },
+      MuiPickersClockPointer: {
+        pointer: {
+          backgroundColor: THEME.SECONDARY,
+        },
+        thumb: {
+          borderColor: THEME.SECONDARY,
+          backgroundColor: 'rgba(254,255, 255, 0.9)',
```

Comment:
Was there a reason to hard-code this value?

Reply:
Um, the `254` in the red-component is actually a typo. It should be `255`. I'll address this in my next commit. 

As for the opacity values (`0.9` and `0.54`), I found these values from the material-ui pickers.dev component demos. They use `0.9` for picker text which is currently selected, while everything else is dimmed out using `0.54`.

Reply:
Ah, ok. We can use `PRIMARY_TEXT_DARK`:

https://github.com/taskcluster/taskcluster/blob/0afbbc46a1cba2d6905c76b5f82392380255f0df/ui/src/utils/constants.js#L48

Reply:
Got it. I'll make this change.